cmd/install/install.go

```diff
@@ -152,7 +152,7 @@ func (o *Options) Validate() error {
 		errs = append(errs, fmt.Errorf("when invoking this command with the --rhobs-monitoring flag, the --enable-cvo-management-cluster-metrics-access flag is not supported "))
 	}
 
-	if len(o.ManagedService) > 0 && o.ManagedService != hyperv1.AroHCP && o.ManagedService != hyperv1.RosaHCP {
```

Comment:
would we add another check for RosaHCP in any other place? or this is not useful anymore?

Reply:
AFAICT as mentioned in the PR desc: there's no usage nor valid semantic for it atm.
This should have never been added. It does nothing.

Reply:
👍 perfect thanks!